### PR TITLE
Bundles Landing Design Tweaks

### PR DESCRIPTION
**Pull request by Ap0c**
## Why are you doing this?

This polishes the design of the landing page, fixing issues with typography, colour scheme, and layout across the breakpoints.

Still a couple of graphical issues in IE and Safari 8, but probably not blockers to going live.

[**Trello Card**](https://trello.com/c/gW9gfqZW/567-polish-design-details-of-the-new-site-in-order-to-go-live)

## Changes

- Updated typography and layout of the introduction text.
- Updated colours, copy and twiddly bits (radio triangle, ticks) for the bundles.
- Fixed bundle layout across the breakpoints.
- Fixed font usage across the page.
- Fixed layout of 'other ways' section.
- Tweaked `.gu-content-margin` to get the right padding.

## Screenshots

To follow.

### CODE REVIEW

File: assets/pages/bundles-landing/bundlesLanding.scss [hunk 1]
```diff
@@ -9,122 +9,114 @@
...
-        margin-right: 20px;
-      }
-
-      @include mq($from: tablet) {
-        margin-left: 260px;
-        margin-bottom: 5px;
-        font-size: 18px;
-        line-height: 22px;
+      &:nth-of-type(1) {
+        margin-left: 80px;
```
Comment (svillafe) [inline assets/pages/bundles-landing/bundlesLanding.scss]: `$gu-h-spacing * 4`

File: assets/pages/bundles-landing/bundlesLanding.scss [hunk 2]
```diff
@@ -9,122 +9,114 @@
...
-        line-height: 28px;
+      &:nth-of-type(2) {
+        margin-left: 120px;
       }
 
-      @include mq($from: leftCol) {
-        margin-left: 420px;
-        margin-right: 100px;
+      &:nth-of-type(3) {
+        margin-left: 40px;
```
Comment (svillafe) [inline assets/pages/bundles-landing/bundlesLanding.scss]: `$gu-h-spacing * 2`

File: assets/pages/bundles-landing/bundlesLanding.scss [hunk 3]
```diff
@@ -400,6 +398,10 @@
       width: 240px;
       margin: 0 ($gu-h-spacing / 2) ($gu-v-spacing * 2) 0;
 
+      @include mq($until: mobileLandscape) {
+
+      }
+
```
Comment (svillafe) [inline assets/pages/bundles-landing/bundlesLanding.scss]: empty media query?

Comment (Ap0c) [inline assets/pages/bundles-landing/bundlesLanding.scss]: ada68e1

**Pull request merged**